src/fshtypes/FshCodeSystem.ts

```diff
@@ -26,51 +23,6 @@ export class FshCodeSystem extends FshEntity {
     return 'FshCodeSystem';
   }
 
-  addConcept(newConcept: ConceptRule) {
-    if (this.checkConcept(newConcept)) {
-      this.rules.push(newConcept);
-    }
-  }
-
-  checkConcept(newConcept: ConceptRule): boolean {
-    const existingConcept = this.rules.find(
-      rule => rule instanceof ConceptRule && rule.code == newConcept.code
-    ) as ConceptRule;
-    if (existingConcept) {
-      // if the new ConceptRule has only a code and a hierarchy,
-      // and the existing ConceptRule with that code has a matching hierarchy,
-      // the user may simply be referencing the existing concept to establish context.
-      if (
-        newConcept.display == null &&
-        newConcept.definition == null &&
-        isEqual(existingConcept.hierarchy, newConcept.hierarchy)
-      ) {
-        return false;
-      } else {
-        throw new CodeSystemDuplicateCodeError(this.id, newConcept.code);
```

Comment:
If we stop throwing `CodeSystemDuplicateCodeError` in the `CodeSystemExporter` then we won't be using it anywhere and we can delete that class.